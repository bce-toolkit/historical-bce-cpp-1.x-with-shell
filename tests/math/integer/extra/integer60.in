2225981351068113943056204993389614240019094595999582111948465524223192357740
7259440577128003225857313916980668997125612557545852240781410277175931113481059361033935824309002280771
