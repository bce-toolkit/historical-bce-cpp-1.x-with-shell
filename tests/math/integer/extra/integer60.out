7259440577128003225857313919206650348193726500602057234171024517195025709480641472982401348532194638511
-7259440577128003225857313914754687646057498614489647247391796037156836517481477249085470300085809923031
16159379343874081442716622491231369426214606369986441773696787695115304083486973916835547390302392576896695002488321980729843082284523451282534409869562694509038264356316955017540
0
2225981351068113943056204993389614240019094595999582111948465524223192357740
0
1
1
1
0
0
